.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: node_js
 
 node_js:
-  - '6'
+  - '8.16.2'
+
+services:
+  - xvfb
 
 branches:
   only:
     - master
 
 before_script:
   - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
+#   - "sh -e /etc/init.d/xvfb start"
```

Comment:
I would like to see this removed if it's no longer needed

Reply:
Doing in the next commit